src/components/Books.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types';
+
+const Books = () => {
+  const books = [
+    {
+      id: 1,
+      title: 'Harry Potter and the Chamber of Secrets',
+      author: 'J.K. Rowling'
+    },
+    {
+      id: 2,
+      title: 'The Originals',
+      author: 'Caroline Kaplin'
+    },
+    {
+      id: 3,
+      title: 'The Firm',
+      author: 'Harvey Specter'
+    },
+    {
+      id: 4,
+      title: 'Game of Thrones',
+      author: 'Damon Salvator'
+    },
+  ];
+
+  return (
+    <div className="Books">
+      <h2>Books</h2>
+      <ul>
+        {books.map(book => {
+          return (<li key={book.id}>{book.title} by {book.author}</li>)
+        })}
+      </ul>
+      <form>
+        <Input type="text" name="Book Title" id="text" key="Book Title" />
+        <Input type="text" name="Book Author" id="text" key="Book Author" />
+        <button type="submit">Add Book</button>
+      </form>
+    </div> 
```

Comment:
- [ ] It will be better to make your components reusable and clear to make the `Books` component contain `<BookList>` which renders a list of books using the `<Book>` component and a `<Book>`:
    * Takes the book data as a prop and renders the title and the author
    * Has a button for removing books